lib/super_digit.rb

```diff
@@ -1,15 +1,26 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(log n)
+# Space Complexity - O(n)- relies on the call stack to pop up
 def super_digit(n)
```

Comment:
Because the loop is O(d) where d is the number of digits and you recur d times, this is O(d^2) in time complexity.